### DISCUSSION TITLE
Oracle DB or Workday connector ?

### DISCUSSION THREAD

**Discussion by divyaswarnkar** [Azure/logicapps#548] [Polls]
Oracle DB or Workday ? Which one should we work next to offer as a [built-in connector](https://docs.microsoft.com/en-us/azure/connectors/compare-built-in-azure-connectors) in Logic Apps.

**Comment by nikolaiblackie** [comment]
We have way more customers using Oracle DBs than Workday APIs. Workday consultants I have dealt with in previous projects are in the stone age and love flat file integrations only as well.

**Comment by numericscloud** [comment]
Workday has already good rest apis and is easy to integrate. 
oracle on the other hand is nightmare.

**Comment by scottrudy** [comment]
The Oracle connector requiring an on premise infrastructure setup is a significant factor for not using logic apps in a hybrid scenario. Having this connector deployed stand alone could increase adoption.

**Comment by divyaswarnkar** [comment]
Thanks for all the feedback. Based on your votes and feedback, Oracle DB seems like the clear winner here. I will keep this thread posted on our plans for it

**Reply by siddheshdp** [reply]
@divyaswarnkar  - Can we get Secured TLS in Oracle built-in and Managed connectors (like MQ connector)? Only basic authentication is not good option. Let me know if there is already a way to establish cert based communication for Oracle and DB2.
